app/templates/frameworks/_framework_actions.html

```diff
@@ -1,5 +1,5 @@
 {% if framework.status in ['standstill', 'live'] and application_made and not countersigned_agreement_file %}
-  {% if not framework.frameworkAgreementVersion %}
+  {% if not framework.frameworkAgreementVersion or not supplier_is_on_framework %}
```

Comment:
Um, couldn't we have `if not (framework.frameworkAgreementVersion and supplier_is_on_framework)` plz? Good old De Morgan, removes any order of precedence ambiguity.